Makefile

```diff
@@ -0,0 +1,204 @@
+.PHONY: all
```

Comment:
FYI for @JeremyTheocharis 
This change comes from the #1548 pr, this is based on.